src/util.js

```diff
@@ -27,10 +27,33 @@ export const camelize = cached((str) => {
   return str.replace(camelizeRE, toUpper)
 })
 
+const kebabizeRE = /([a-z0-9])([A-Z])/g
+export const kebabize = cached((str) => {
+  console.log('kebabize', str)
```

Comment:
This line should probably not be a part of it?

Reply:
Good catch!